### PR TITLE
fix #1852, update sbt to 0.13.9

### LINKED ISSUE #1852: update sbt to 0.13.9
It has bunch of fixes in cache resolution,
https://github.com/sbt/sbt/releases

### PR DESCRIPTION
**Pull request by clockfly**
The packaging and cross-build looks good in local test.

### CODE REVIEW

**Comment by huafengw** [comment]
```
Downloading sbt launcher for 0.13.9:
  From  http://typesafe.artifactoryonline.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/0.13.9/sbt-launch.jar
    To  /home/travis/.sbt/launchers/0.13.9/sbt-launch.jar
Download failed. Obtain the jar manually and place it at /home/travis/.sbt/launchers/0.13.9/sbt-launch.jar
```

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@ language:
 - java
 - scala
 sudo: false
+before_script:
```
Comment (stanleyxu2005) [inline .travis.yml]: Travis will not take care of downloading a suitable sbt-launcher?

Comment (clockfly) [inline .travis.yml]: Yes, that is a bug of travis or sbt repo.

**Comment by manuzhang** [comment]
+1

File: streaming/src/test/scala/io/gearpump/streaming/appmaster/AppMasterSpec.scala [hunk 2]
```diff
@@ -203,7 +203,7 @@ object AppMasterSpec {
   val MASTER = "master"
   case object TaskStarted
 
-  val MOCK_MASTER_PROXY = "mockMasterProxy"
+  def masterProxyName = "mockMasterProxy"
```
Comment (manuzhang) [inline streaming/src/test/scala/io/gearpump/streaming/appmaster/AppMasterSpec.scala]: why changed this ?

**Pull request closed without merging**